.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -59,12 +59,12 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
     - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
     - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
     - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
-- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
+- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers. Check this off if it does not apply.
 - [ ] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
-- [ ] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
+- [ ] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected). Check this off if it does not apply.
 - [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
 - [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
-- [ ] If a new component is created I verified that:
+- [ ] If a new component is created I verified that (check these off if they do not apply):
```

Comment:
hmm, seems you have two different patterns for the same thing:
1. `. Check this off if it does not apply.`
2. `(check these off if they do not apply)` 

Not sure why we can't use `(check these off if they do not apply)` everywhere
And on the items that have subitems, it's not clear to me if we should only check the parent item, or the parent and all of the children.